From: Michael Biebl <biebl@debian.org>
Date: Thu, 2 Jun 2016 17:41:29 +0200
Subject: systemd: Add Documentation to service files

Reference the nm-online and NetworkManager man page in the corresponding
service files.

https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Documentation=

https://bugzilla.gnome.org/show_bug.cgi?id=767168
---
 data/NetworkManager-wait-online-systemd-pre200.service.in | 1 +
 data/NetworkManager-wait-online.service.in                | 1 +
 data/NetworkManager.service.in                            | 1 +
 3 files changed, 3 insertions(+)

diff --git a/data/NetworkManager-wait-online-systemd-pre200.service.in b/data/NetworkManager-wait-online-systemd-pre200.service.in
index d145a79..8b54f48 100644
--- a/data/NetworkManager-wait-online-systemd-pre200.service.in
+++ b/data/NetworkManager-wait-online-systemd-pre200.service.in
@@ -1,5 +1,6 @@
 [Unit]
 Description=Network Manager Wait Online
+Documentation=man:nm-online(1)
 Requisite=NetworkManager.service
 After=NetworkManager.service
 Wants=network.target
diff --git a/data/NetworkManager-wait-online.service.in b/data/NetworkManager-wait-online.service.in
index 8c4fa67..4df854c 100644
--- a/data/NetworkManager-wait-online.service.in
+++ b/data/NetworkManager-wait-online.service.in
@@ -1,5 +1,6 @@
 [Unit]
 Description=Network Manager Wait Online
+Documentation=man:nm-online(1)
 Requisite=NetworkManager.service
 After=NetworkManager.service
 Before=network-online.target
diff --git a/data/NetworkManager.service.in b/data/NetworkManager.service.in
index eaab677..4f07180 100644
--- a/data/NetworkManager.service.in
+++ b/data/NetworkManager.service.in
@@ -1,5 +1,6 @@
 [Unit]
 Description=Network Manager
+Documentation=man:NetworkManager(8)
 Wants=network.target
 After=network-pre.target dbus.service
 Before=network.target @DISTRO_NETWORK_SERVICE@
